>From 2eca23678ddaad0d7f7df2ce79868e27651a96d8 Mon Sep 17 00:00:00 2001
From: gregor herrmann <gregoa@debian.org>
Date: Tue, 4 Feb 2014 17:42:57 +0100
Subject: [PATCH] Use realclean instead of distclean on perl_build buildsystem.

Switch from `Build distclean' to `Build realclean' in the clean action of
the perl_build buildsystem.

'distclean' is not specified in the CPAN::API::BuildPL specification, and
therefor not implemented in Module::Build::Tiny (and other future
implementations). Besides that, the dist* actions are meant for upstream
authors.
---
 Debian/Debhelper/Buildsystem/perl_build.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Debian/Debhelper/Buildsystem/perl_build.pm b/Debian/Debhelper/Buildsystem/perl_build.pm
index bfe1c08..4cfbcb4 100644
--- a/Debian/Debhelper/Buildsystem/perl_build.pm
+++ b/Debian/Debhelper/Buildsystem/perl_build.pm
@@ -70,7 +70,7 @@ sub install {
 sub clean {
 	my $this=shift;
 	if (-e $this->get_sourcepath("Build")) {
-		$this->do_perl("Build", "distclean", "--allow_mb_mismatch", 1, @_);
+		$this->do_perl("Build", "realclean", "--allow_mb_mismatch", 1, @_);
 	}
 }
 
-- 
1.9.rc1

